NOTICE

```diff
@@ -0,0 +1,9 @@
+Apache Flink
+Copyright 2014-2020 The Apache Software Foundation
+
+This product includes software developed at
+The Apache Software Foundation (http://www.apache.org/).
+
+This project contains copied and modified source code that is licensed under the Apache Software License 2.0 (http://www.apache.org/licenses/LICENSE-2.0.txt) from the following projects:
+
+- fastutil (http://fastutil.di.unimi.it/) Copyright (c) - Sebastiano Vigna
```

Comment:
Are the files in question tagged appropriately? For maintenance purposes you always want to be able to trace down the files from the NOTICE.

Reply:
@zentol what kind of tagging do you have in mind?
Would a mention in NOTICE of where the copied files are in the source be sufficient?

Reply:
A small note in the javadocs + paths in the notice, to cover all our bases.

Reply:
Got it, thanks